core/src/main/java/org/elasticsearch/gateway/MetaDataStateFormat.java

```diff
@@ -270,6 +267,8 @@ public  T loadLatestState(ESLogger logger, Path... dataLocations) throws IOExcep
                             files.add(pav);
                         }
                     }
+                } catch (NoSuchFileException | FileNotFoundException ex) {
+                    // no _state directory -- move on
```

Comment:
Further down in the function it looks for the file and appropriately errors out if they were to be missing, I don't think this will be an issue.